nemo_text_processing/inverse_text_normalization/fr/data/roman/digits_large.tsv

```diff
@@ -0,0 +1,9 @@
+I	1
+II	2
+III	3
```

Comment:
Where are these `roman` files used? I couldn't find a RomanFst

Reply:
They're used for the ordinal verbalizer. Since it's only this one case it seemed to make sense just to do it in the verbalization phase rather than add a new fst into the mix.